shell/platform/tizen/flutter_tizen_view.cc

```diff
@@ -0,0 +1,303 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter_tizen_view.h"
+
+#include "flutter/shell/platform/tizen/logger.h"
+#include "flutter/shell/platform/tizen/tizen_window.h"
+
+namespace {
+
+#if defined(MOBILE_PROFILE)
+constexpr double kProfileFactor = 0.7;
+#elif defined(WEARABLE_PROFILE)
+constexpr double kProfileFactor = 0.4;
+#elif defined(TV_PROFILE)
+constexpr double kProfileFactor = 2.0;
+#else
+constexpr double kProfileFactor = 1.0;
+#endif
+
+constexpr char kBackKey[] = "XF86Back";
+constexpr char kExitKey[] = "XF86Exit";
+
+// Keys that should always be handled by the app first but not by the system.
+const std::vector<std::string> kBindableSystemKeys = {
```

Comment:
This constant is not being used.

Where do you think `BindKeys` should be called? In this class (in the constructor)?

Reply:
Exactly! This is my mistake. Thank you for finding this out.